WordPress/Classes/ViewRelated/Plans/PlanPostPurchaseViewController.swift

```diff
@@ -328,17 +328,28 @@ class PlanPostPurchasePageViewController: UIViewController {
     }
 }
 
-enum PlanPostPurchasePageType: Int {
-    case PurchaseComplete
-    case Customize
-    case VideoPress
-    case Themes
+enum PlanPostPurchasePageType: String {
+    case PurchaseComplete = "purchase-complete"
+    case Customize = "custom-design"
+    case VideoPress = "videopress"
+    case Themes = "premium-themes"
+    
+    // This is the order we'd like pages to appear in the post purchase flow
+    static var orderedPageTypes: [PlanPostPurchasePageType] { return [ .PurchaseComplete, .Customize, .VideoPress, .Themes ] }
     
     static func pageTypesForPlan(plan: Plan) -> [PlanPostPurchasePageType] {
-        switch plan.slug {
-        case "premium": return [.PurchaseComplete, .Customize, .VideoPress]
-        case "business": return [.PurchaseComplete, .Customize, .VideoPress, .Themes]
-        default: return []
+        var types = [ PlanPostPurchasePageType.PurchaseComplete ]
+        
+        // Get all of the page types for the plan's features
+        let slugs = plan.featureGroups.flatMap { $0.slugs }.flatMap { PlanPostPurchasePageType(rawValue: $0) }
```

Comment:
I don't think we have discussed this, but for trailing closures I like to:

- Put each operation on a new line
- Use parenthesis if it's functional (e.g. `map`), don't use them if procedural (e.g. `forEach`)

In this case:

```swift
let slugs = plan.featureGroups
    .flatMap({ $0.slugs })
    .flatMap({ PlanPostPurchasePageType(rawValue: $0) })
```

Aslo, for the second `flatMap` you can pass the initializer directly:

```swift
let slugs = plan.featureGroups
    .flatMap({ $0.slugs })
    .flatMap(PlanPostPurchasePageType.init)
```